cmd/kops/create_cluster.go

```diff
@@ -533,6 +536,10 @@ func RunCreateCluster(f *util.Factory, out io.Writer, c *CreateClusterOptions) e
 		cluster.Spec.CloudProvider = c.Cloud
 
 		if c.Cloud == "vsphere" {
+			if !featureflag.VSphereCloudProvider.Enabled() {
+				return fmt.Errorf("Cloud vSphere is not supported.")
```

Comment:
How about- 'Feature flag VSphereCloudProvider is not set. Cloud vSphere will not be supported.'